creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -69,6 +74,16 @@ class SnapbackSM {
       discoveryProviderEndpoint: audiusLibs.discoveryProvider.discoveryProviderEndpoint,
       creatorNodeEndpoint: this.endpoint
     })
+
+    // Config to determine if reconfig is enabled
+    this.snapbackReconfigEnabled = (typeof this.audiusLibs.snapbackDevModeEnabled === 'boolean' && this.audiusLibs.snapbackDevModeEnabled) ||
+      this.nodeConfig.get('snapbackReconfigEnabled')
+
+    // The interval when SnapbackSM is fired for state machine jobs
+    this.snapbackJobInterval = this.nodeConfig.get('snapbackJobInterval') // ms
+
+    // Array of healthy node endpoints returned from CreatorNodeSelection logic
+    this.healthyNodes = null
```

Comment:
is there any value in setting this as a class var if it is never persisted? i see you clear it later but feels like you could just remove both and pass the value to the function after it is calculated